### PR TITLE
[Feature] Support YOLOv5 deployment at RV1126 device

### PR DESCRIPTION
**Pull request by AllentDan**

### CODE REVIEW

File: configs/deploy/detection_rknn_static-320x320.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+_base_ = ['./base_static.py']
+onnx_config = dict(
+    input_shape=[320, 320], output_names=['feat0', 'feat1', 'feat2'])
+use_efficientnms = False
```
Comment (triple-Mu) [inline configs/deploy/detection_rknn_static-320x320.py:4]:
```suggestion
```

File: mmyolo/deploy/models/dense_heads/yolov5_head.py [hunk 2]
```diff
@@ -146,3 +146,33 @@ def yolov5_head__predict_by_feat(ctx,
 
     return nms_func(bboxes, scores, max_output_boxes_per_class, iou_threshold,
                     score_threshold, pre_top_k, keep_top_k)
+
+
+@FUNCTION_REWRITER.register_rewriter(
+    func_name='mmyolo.models.dense_heads.yolov5_head.'
+    'YOLOv5Head.predict',
+    backend='rknn')
+def yolov5_head__predict__rknn(ctx, self, x: Tuple[Tensor], *args, **kwargs):
```
Comment (PeterH0323) [inline mmyolo/deploy/models/dense_heads/yolov5_head.py:155]:
```suggestion
def yolov5_head__predict__rknn(ctx, self, x: Tuple[Tensor], *args, **kwargs) -> Tensor:
```

File: mmyolo/deploy/models/dense_heads/yolov5_head.py [hunk 3]
```diff
@@ -146,3 +146,34 @@ def yolov5_head__predict_by_feat(ctx,
...
     return nms_func(bboxes, scores, max_output_boxes_per_class, iou_threshold,
                     score_threshold, pre_top_k, keep_top_k)
+
+
+@FUNCTION_REWRITER.register_rewriter(
+    func_name='mmyolo.models.dense_heads.yolov5_head.'
+    'YOLOv5Head.predict',
+    backend='rknn')
+def yolov5_head__predict__rknn(ctx, self, x: Tuple[Tensor], *args, **kwargs) -> Tensor:
+                               **kwargs) -> Tuple[Tensor, Tensor, Tensor]:
```
Comment (PeterH0323) [inline mmyolo/deploy/models/dense_heads/yolov5_head.py:156]: maybe something wrong after the merge 😢

Comment (AllentDan) [reply mmyolo/deploy/models/dense_heads/yolov5_head.py]: Not familiar with `Allow edits by maintainers`, closed now.

**Comment by triple-Mu** [comment]
LGTM

**Review by triple-Mu — APPROVED** [review]
LGTM

**Review by PeterH0323 — APPROVED** [review]
LGTM

**Comment by PeterH0323** [comment]
Hi @AllentDan 

Pls fix the ci error

**Review by hhaAndroid — APPROVED** [review]

**Pull request merged**